Tests/fixtures/smartphone-18.yml

```diff
@@ -4561,8 +4561,8 @@
     type: browser
     name: Aloha Browser
     version: 2.20.3
-    engine: WebKit
-    engine_version: "537.36"
+    engine: Blink
+    engine_version: ""    
```

Comment:
end 4 spaces should be removed 'engine_version: ""
'

Reply:
Done